pkg/rbd/controllerserver.go

```diff
@@ -122,7 +128,7 @@ func (cs *ControllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 		return nil, err
 	}
 
-	found, err := checkVolExists(rbdVol, req.GetSecrets())
+	found, err := checkVolExists(rbdVol, cr.ID, cr.Key)
```

Comment:
pass `cr` struct to `checkVolExists`

Reply:
same applies wherever we are passing `ID and Key`

Reply:
Done.

Did this more across the code, so that it is cleaner, than just certain instances.